lib/jmap/mail/vacation/vacation_response.dart

```diff
@@ -24,7 +24,7 @@ class VacationResponse with EquatableMixin {
 
   final String? subject;
 
-  @JsonKey(includeIfNull: false)
+  @JsonKey(includeIfNull: true)
```

Comment:
IMO, It's default value is true. So we only remove this line.

Reply:
Done